v2/README.md

```diff
@@ -60,9 +53,7 @@ type CallOptions func(i interface{})
 type Store interface {
 	NewTransaction(ctx context.Context, records ...Record) (StoreTransaction, error)
 	LoadByAggregate(ctx context.Context, aggregateID string, opts ...QueryOptions) (record []Record, err error)
-	LoadBySequenceID(ctx context.Context, sequenceID string, opts ...QueryOptions) (record []Record, err error)
-	LoadBySequenceIDAndType(ctx context.Context, sequenceID string, eventType string, opts ...QueryOptions) (records []Record, err error)
```

Comment:
a drawback may be that the query options feels a bit SQL like

Reply:
I added both Equals and BySequenceID(..). The first one is very lookalike a SQL builder, but the second one is not. Just to make it more flexiable. I can remove the Equals / LessThan / GreaterThan exports, so you just get the By* functions

Reply:
I no longer exports the LessThan / GreaterThan or Equals methods :) Less SQL Builder like